src/ng/httpBackend.js

```diff
@@ -109,7 +109,12 @@ function createHttpBackend($browser, createXhr, $browserDefer, callbacks, rawDoc
         }
       }
 
-      xhr.send(post || null);
+      if (post === false) {
```

Comment:
That would be ideal. See https://github.com/angular/angular.js/issues/11552. Unfortunately Travis CI E2E tests failed in that case. If this fails, I'll dig deeper since it implies strongly that something is wrong with the tests. This is my first attempt at contributing to angular, so perhaps I don't understand the intended workflow well yet. I've been unable to build/test on my local machine per these instructions: https://docs.angularjs.org/misc/contribute.